packages/botonic-plugin-contentful/jest.config.js

```diff
@@ -1,6 +1,6 @@
 // Options about JS are for compiling @botonic .js/jsx files
```

Comment:
I see there's no jest.config.js for core.
Can you add one with minimal contents? Not sure if by default it was already adding src to the roots

Reply:
good catch @dpinol ! There were some files missing from src.
I added a jest config file for taking care of that.